modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/ComponentMessageProcessorClassLoadingTestCase.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.internal.runtime;
+
+import static org.mule.runtime.core.api.lifecycle.LifecycleUtils.*;
+import static org.mule.runtime.core.api.lifecycle.LifecycleUtils.disposeIfNeeded;
+import static org.mule.runtime.oauth.internal.util.ClassLoaderUtils.setContextClassLoader;
+
+import static java.lang.Thread.currentThread;
+import static java.util.Optional.of;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.sameInstance;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.withSettings;
+
+import org.mule.runtime.api.exception.MuleException;
+import org.mule.runtime.api.meta.model.ComponentModel;
+import org.mule.runtime.api.meta.model.EnrichableModel;
+import org.mule.runtime.api.meta.model.ExtensionModel;
+import org.mule.runtime.api.meta.model.XmlDslModel;
+import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.core.api.extension.ExtensionManager;
+import org.mule.runtime.core.internal.metadata.cache.MetadataCacheIdGeneratorFactory;
+import org.mule.runtime.core.internal.policy.PolicyManager;
+import org.mule.runtime.core.privileged.processor.chain.DefaultMessageProcessorChainBuilder;
+import org.mule.runtime.core.privileged.processor.chain.MessageProcessorChain;
+import org.mule.runtime.extension.api.runtime.config.ConfigurationProvider;
+import org.mule.runtime.module.extension.api.loader.java.property.CompletableComponentExecutorModelProperty;
+import org.mule.runtime.module.extension.internal.runtime.operation.ComponentMessageProcessor;
+import org.mule.runtime.module.extension.internal.runtime.resolver.ResolverSet;
+import org.mule.tck.junit4.AbstractMuleContextTestCase;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Issue;
+import org.junit.After;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ComponentMessageProcessorClassLoadingTestCase extends AbstractMuleContextTestCase {
```

Comment:
allure annotations.

Reply:
Which allure annotations should I add here? I added the `Description` and the `Issue` annotations to the test method.